DllImportGenerator/DllImportGenerator/Marshalling/BlittableArrayMarshaller.cs

```diff
@@ -0,0 +1,232 @@
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.Interop
+{
+    internal class BlittableArrayMarshaller : ConditionalStackallocMarshallingGenerator
+    {
+        private const int StackAllocBytesThreshold = 0x200;
```

Comment:
Document why we chose this number. If it was just a "reasonable guess" that is fine, but we need to know why.